### PR TITLE
71 conecting tasktree UI to backend

### LINKED ISSUE #71: [Feature 12] Connecting TaskTree UI to Backend
Creating a functional TaskTree UI by connecting it to the backend functionality.

### PR DESCRIPTION
**Pull request by Bhan-Aayush**
I was able to complete my input boundary, Interactor, Controller and display model. The TaskTree UI, Interface, Presenter and Output boundary are still a work in progress.

### CODE REVIEW

File: src/main/java/task_tree_UI/TaskTreeUI.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+/**
+ * A TaskTree UI class, that implements the JFrame interface by making a list of tasks displayed by a tree.
+ * Created: 11/15/2022
+ * Last updated: 11/26/2022
+ *
+ * @author Aayush Bhan
+ */
+
+
+public class TaskTreeUI extends CentralWindow {
```
Comment (MMachadoUofT) [inline src/main/java/task_tree_UI/TaskTreeUI.java:26]: Good work slimming down here; while the file "looks" a little emptier, this definitely adheres more to the clean architecture, and keeps your code open for when you eventually implement your presenter and models.

File: src/main/java/use_cases/display_task_tree/TaskTreeUIInterface.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package use_cases.display_task_tree;
+
+public interface TaskTreeUIInterface {
```
Comment (MMachadoUofT) [inline src/main/java/use_cases/display_task_tree/TaskTreeUIInterface.java:3]: Make sure they you end up having your TaskTreeUI implement this interface! This is going to be the key in presenting your TaskTreeDisplayModel to the UI.

**Review by MMachadoUofT — APPROVED** [review]
This is great work! I know that you've been struggling a bit with how to implement a tree structure within JFrame, and I don't blame you; there's a lot of functionality to be implemented. I do, however, believe this is an excellent start; for the moment, some next steps should involve figuring out how to add action listeners to your tree nodes. Perhaps make an unversioned file that you can experiment a little in? The more you know and sooner, the better.

File: src/main/java/task_tree_UI/TaskTreeUI.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        // set Frame window size
+        this.setSize(350, 300);
+        // disable resizability
+        this.setResizable(false);
+        // set content, configured in form file
+        this.setContentPane(mainPanel);
+    }
+}
+
+
```
Comment (Amir1441) [inline src/main/java/task_tree_UI/TaskTreeUI.java:57]: Nice work here, the next step is to incorporate the Window Manager interface which will allow for all windows to be created on run. If you want a reference you can look at David's Login UI and create account UI which I too used as a reference when including the Window manager interface to UI

**Review by Amir1441 — APPROVED** [review]
There is a lot of technicality that comes with what you are doing especially with the front end work, dealing with trees in JFrame. That being said, you're progressing well. More functionality can be added to the UI side as well remembering to incorporate that Window Manager interface, as it is essential in the long run

**Review by bmguiler — APPROVED** [review]

**Pull request merged**